11-Aug-2020 17:53:08.805 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 17:53:08.805 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 17:53:08.820 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@515f4131')
11-Aug-2020 17:56:28.474 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 17:56:28.474 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 17:56:35.307 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 17:56:35.307 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 17:56:35.309 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@21fff664')
11-Aug-2020 18:06:58.498 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 18:06:58.498 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 18:07:05.550 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 18:07:05.551 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 18:07:05.553 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@1e6308a9')
11-Aug-2020 18:09:43.147 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 18:09:43.147 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 18:09:49.850 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 18:09:49.850 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 18:09:49.850 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@245a060f')
11-Aug-2020 18:29:02.678 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 18:29:02.678 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 18:29:09.495 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 18:29:09.495 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 18:29:09.498 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@21ab988f')
11-Aug-2020 18:29:36.143 SEVERE [http-nio-8080-exec-5] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:487)
		at org.apache.catalina.startup.HostConfig.check(HostConfig.java:1642)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:288)
		at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:809)
		at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
		at org.apache.catalina.manager.ManagerServlet.check(ManagerServlet.java:1593)
		at org.apache.catalina.manager.ManagerServlet.deploy(ManagerServlet.java:1022)
		at org.apache.catalina.manager.ManagerServlet.doGet(ManagerServlet.java:355)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:626)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:666)
		at org.apache.catalina.valves.RequestFilterValve.process(RequestFilterValve.java:348)
		at org.apache.catalina.valves.RemoteAddrValve.invoke(RemoteAddrValve.java:53)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 46 more
11-Aug-2020 18:29:45.127 SEVERE [http-nio-8080-exec-10] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:29:45.128 SEVERE [http-nio-8080-exec-10] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:29:49.921 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:29:49.921 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:30.230 SEVERE [http-nio-8080-exec-16] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:30.230 SEVERE [http-nio-8080-exec-16] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:40.949 SEVERE [http-nio-8080-exec-18] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:40.949 SEVERE [http-nio-8080-exec-18] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:46.251 SEVERE [http-nio-8080-exec-13] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:30:46.252 SEVERE [http-nio-8080-exec-13] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 18:38:34.800 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 18:38:34.800 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 18:39:53.346 SEVERE [main] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	java.sql.SQLException: mariadb-java-client-2.6.2
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1830)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:118)
		at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:526)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:425)
		at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576)
		at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309)
		at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
		at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423)
		at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936)
		at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:841)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
		at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:738)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
	Caused by: java.lang.ClassNotFoundException: mariadb-java-client-2.6.2
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 42 more
11-Aug-2020 18:39:55.870 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 18:39:55.871 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 18:39:55.878 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@6dce59e')
11-Aug-2020 19:18:58.615 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 19:18:58.615 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 19:23:03.188 SEVERE [main] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1830)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:118)
		at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:526)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:425)
		at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576)
		at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309)
		at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
		at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423)
		at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936)
		at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:841)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
		at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:738)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 42 more
11-Aug-2020 19:23:04.021 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 19:23:04.021 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 19:23:04.023 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@63c5efee')
11-Aug-2020 19:23:43.541 SEVERE [http-nio-8080-exec-8] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:487)
		at org.apache.catalina.startup.HostConfig.check(HostConfig.java:1642)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.tomcat.util.modeler.BaseModelMBean.invoke(BaseModelMBean.java:288)
		at java.management/com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:809)
		at java.management/com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
		at org.apache.catalina.manager.ManagerServlet.check(ManagerServlet.java:1593)
		at org.apache.catalina.manager.ManagerServlet.deploy(ManagerServlet.java:1022)
		at org.apache.catalina.manager.ManagerServlet.doGet(ManagerServlet.java:355)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:626)
		at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
		at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
		at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
		at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
		at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:666)
		at org.apache.catalina.valves.RequestFilterValve.process(RequestFilterValve.java:348)
		at org.apache.catalina.valves.RemoteAddrValve.invoke(RemoteAddrValve.java:53)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 46 more
11-Aug-2020 19:23:52.943 SEVERE [http-nio-8080-exec-2] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:23:52.943 SEVERE [http-nio-8080-exec-2] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:24:00.367 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:24:00.368 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:24:18.558 SEVERE [http-nio-8080-exec-15] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:24:18.558 SEVERE [http-nio-8080-exec-15] org.apache.catalina.realm.JDBCRealm.authenticate Excepci贸n al realizar la autentificaci贸n
	java.sql.SQLException: com.mysql.jdbc.Driver
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:644)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:317)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
	Caused by: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
		at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:435)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:589)
		at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
		at java.base/java.lang.Class.forName0(Native Method)
		at java.base/java.lang.Class.forName(Class.java:340)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:640)
		... 17 more
11-Aug-2020 19:56:10.068 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 19:56:10.068 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 19:56:31.682 SEVERE [main] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'basededatos'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:864)
		at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1707)
		at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1217)
		at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2189)
		at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2220)
		at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2015)
		at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:768)
		at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:385)
		at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:323)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:654)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1830)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:118)
		at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:526)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:425)
		at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576)
		at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309)
		at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
		at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423)
		at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936)
		at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:841)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
		at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:738)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
11-Aug-2020 19:56:33.731 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 19:56:33.731 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 19:56:33.735 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@4f0cab0a')
11-Aug-2020 20:02:35.726 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:02:35.726 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:02:44.572 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:02:44.572 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:02:44.574 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@2e23c180')
11-Aug-2020 20:04:38.898 SEVERE [http-nio-8080-exec-14] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:04:38.903 SEVERE [http-nio-8080-exec-14] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:04:45.232 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:04:45.238 SEVERE [http-nio-8080-exec-7] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:04:53.383 SEVERE [http-nio-8080-exec-4] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:04:53.387 SEVERE [http-nio-8080-exec-4] org.apache.catalina.realm.JDBCRealm.getPassword Excepci贸n al realizar la autentificaci贸n
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2524)
		at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2675)
		at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2465)
		at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1915)
		at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:2023)
		at org.apache.catalina.realm.JDBCRealm.getPassword(JDBCRealm.java:522)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:377)
		at org.apache.catalina.realm.JDBCRealm.authenticate(JDBCRealm.java:320)
		at org.apache.catalina.authenticator.FormAuthenticator.doAuthenticate(FormAuthenticator.java:243)
		at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:631)
		at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
		at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
		at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
		at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
		at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
		at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
		at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
		at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
		at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1589)
		at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
		at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
		at java.base/java.lang.Thread.run(Thread.java:832)
11-Aug-2020 20:15:19.797 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:15:19.797 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:31:56.974 SEVERE [main] org.apache.catalina.realm.JDBCRealm.startInternal Excepci贸n abriendo la conexi贸n a la base de datos
	com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'server'
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.Util.getInstance(Util.java:386)
		at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:944)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3933)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3869)
		at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:864)
		at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1707)
		at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1217)
		at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2189)
		at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2220)
		at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2015)
		at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:768)
		at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
		at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
		at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
		at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
		at com.mysql.jdbc.Util.handleNewInstance(Util.java:403)
		at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:385)
		at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:323)
		at org.apache.catalina.realm.JDBCRealm.open(JDBCRealm.java:654)
		at org.apache.catalina.realm.JDBCRealm.startInternal(JDBCRealm.java:715)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5023)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:717)
		at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:690)
		at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:705)
		at org.apache.catalina.startup.HostConfig.deployDescriptor(HostConfig.java:631)
		at org.apache.catalina.startup.HostConfig$DeployDescriptor.run(HostConfig.java:1830)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:118)
		at org.apache.catalina.startup.HostConfig.deployDescriptors(HostConfig.java:526)
		at org.apache.catalina.startup.HostConfig.deployApps(HostConfig.java:425)
		at org.apache.catalina.startup.HostConfig.start(HostConfig.java:1576)
		at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:309)
		at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
		at org.apache.catalina.util.LifecycleBase.setStateInternal(LifecycleBase.java:423)
		at org.apache.catalina.util.LifecycleBase.setState(LifecycleBase.java:366)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:936)
		at org.apache.catalina.core.StandardHost.startInternal(StandardHost.java:841)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1384)
		at org.apache.catalina.core.ContainerBase$StartChild.call(ContainerBase.java:1374)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at org.apache.tomcat.util.threads.InlineExecutorService.execute(InlineExecutorService.java:75)
		at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:140)
		at org.apache.catalina.core.ContainerBase.startInternal(ContainerBase.java:909)
		at org.apache.catalina.core.StandardEngine.startInternal(StandardEngine.java:262)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardService.startInternal(StandardService.java:421)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:930)
		at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:738)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
11-Aug-2020 20:31:59.207 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:31:59.207 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:31:59.213 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@76f856a8')
11-Aug-2020 20:39:50.024 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:39:50.024 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:39:57.613 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:39:57.613 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:39:57.615 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@53dad875')
11-Aug-2020 20:41:53.970 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:41:53.970 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:42:01.355 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:42:01.355 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:42:01.359 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@2e23c180')
11-Aug-2020 20:49:12.395 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:49:12.395 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:49:19.896 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:49:19.896 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:49:19.899 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@22590e3e')
11-Aug-2020 20:54:33.714 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 20:54:33.714 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
11-Aug-2020 20:55:49.519 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextInitialized()
11-Aug-2020 20:55:49.519 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextInitialized()
11-Aug-2020 20:55:49.521 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: attributeAdded('StockTicker', 'async.Stockticker@59018eed')
11-Aug-2020 21:14:00.355 INFO [main] org.apache.catalina.core.ApplicationContext.log SessionListener: contextDestroyed()
11-Aug-2020 21:14:00.356 INFO [main] org.apache.catalina.core.ApplicationContext.log ContextListener: contextDestroyed()
